.gitignore

```diff
@@ -108,3 +108,9 @@ derby.log
 /extide/gradle/netbeans-gradle-tooling/gradle/wrapper/gradle-wrapper.jar
 /extide/gradle/release/modules/gradle/daemon-loader/.gradle/
 /nbbuild/misc/prepare-bundles/target/
+
+# ANTLR v4 Grammar
+##################################
+/java/languages.antlr/external/*.g4
+/java/languages.antlr/external/LexerAdaptor.java
```

Comment:
Why are these files downloaded at runtime? If I'm not mistaken (3-Clause BSD), we can hold them in the repository and thus don't need a build-time download (external repositories will go away)

Reply:
Thanks! That's the wisdom I came to seek!